### PR TITLE
Fix RSS validate error

### PR DESCRIPTION
**Pull request by netroby**
see https://github.com/gorilla/feeds/issues/26

### CODE REVIEW

File: rss.go [hunk 1]
```diff
@@ -108,8 +108,8 @@ func newRssItem(i *Item) *RssItem {
 
 // create a new RssFeed with a generic Feed struct's data
 func (r *Rss) RssFeed() *RssFeed {
-	pub := anyTimeFormat(time.RFC822, r.Created, r.Updated)
-	build := anyTimeFormat(time.RFC822, r.Updated)
+	pub := anyTimeFormat(time.RFC1123Z, r.Created, r.Updated)
+	build := anyTimeFormat(time.RFC1123Z, r.Updated)
```
Comment (elithrar) [inline rss.go]: The original issue/W3C validator requires an RFC822 time as per https://github.com/gorilla/feeds/issues/26

I'm confused as to why we're changing it to RFC1123Z?

**Comment by elithrar** [comment]
Hi @netroby - see my comment.

**Comment by netroby** [comment]
see the w3c validate result



通过 Android 版 Outlook 发送






On Sun, Nov 22, 2015 at 1:10 AM -0800, "Matt Silverlock" <notifications@github.com> wrote:












Hi @netroby - see my comment.

**Comment by netroby** [comment]
https://validator.w3.org/feed/docs/error/InvalidRFC2822Date.html

**Comment by netroby** [comment]
The golang powered RFC822 time format won't pass w3c rss validator

**Comment by elithrar** [comment]
Alright, thanks for the link. This is because the RSS validator/specification wants a non-standard RFC822 date with a 4-digit year, which (to be precise) **not** an RFC822 date (only 2-digit years). Go is 'correct' here, at the expense of not being compatible with the RSS standard.

I'm not sure whether this is a problem beyond the W3C validator, but I don't see the fix as breaking anything.

You'll need to update the tests with the corrected RFC1123Z dates as part of this PR.

**Comment by netroby** [comment]
W3C is the standard. may be they were wrong. but we can not change it. so we should adjust our code to match the standard

**Comment by netroby** [comment]
Done, finish the test, and passed the check

**Comment by elithrar** [comment]
Merged. Thanks for contributing this @netroby!

**Comment by netroby** [comment]
My honor , sir

**Pull request merged**